releasing/plugin-commands-deploy/src/createDeployFiles.ts

```diff
@@ -112,7 +113,9 @@ export function createDeployFiles ({
     const inputDependencies = inputSnapshot[field] ?? {}
     for (const name in inputDependencies) {
       const spec = inputDependencies[name]
-      const resolveResult = resolveLinkOrFile(spec, {
+      const depPath = `${name}@${spec}` as DepPath
```

Comment:
This is correct but you need to use "version" instead of "spec"

Reply:
This value can be `pkg@ver(peersSuffix)` or `pkg@file:path(peersSuffix)`. Is it right to rename it to `version`?